src/css/App.css

```diff
@@ -899,6 +897,490 @@ a:hover {
     background-color: #e6e4e4;
 }
 
+.clinical-panel-container {
```

Comment:
I think you could moving styles related to the clinical panel to a separate CSS file, e.g., css/ClinicalPanel.css